pkg/debugapi/node.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2021 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package debugapi
+
+import (
+	"net/http"
+
+	"github.com/ethersphere/bee/pkg/jsonhttp"
+)
+
+type BeeNodeMode uint
+
+const (
+	LightMode BeeNodeMode = iota
+	FullMode
+	DevMode
+)
+
+type nodeResponse struct {
+	BeeMode     string `json:"beeMode"`
+	GatewayMode bool   `json:"gatewayMode"`
+}
+
+func (b BeeNodeMode) String() string {
```

Comment:
It would be good to add a test for this. You can look at [this](https://github.com/ethersphere/bee/blob/39cda82e11d57d3146ea53e8ac7bc2b8e44080a2/pkg/p2p/p2p_test.go#L23) for examples.